superset/views/core.py

```diff
@@ -15,12 +15,13 @@
 from flask_appbuilder.actions import action
```

Comment:
Could you move this its own .py file. `core.py` is getting really big and need to start breaking down the functionality.

Reply:
Responding to a similar comment below